index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
 <h5 id="wide-review">
 Wide Review</h5>
 
-	The requirements for <dfn id=dfn-wide-review export>wide review</dfn> are not precisely defined by the W3C Process.
-	The objective is to ensure that the entire set of stakeholders of the Web community,
-	including the general public,
-	have had adequate notice of the progress of the [=Working Group=]
-	(for example through notices posted to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>)
-	and were able to actually perform reviews of and provide comments on the specification.
-	A second objective is to encourage groups to request reviews
-	early enough that comments and suggested changes
-	can still be reasonably incorporated in response to the review.
-	Before approving transitions,
-	the [=Director=] will consider who has been explicitly offered
-	a reasonable opportunity to review the document,
-	who has provided comments,
-	the record of requests to and responses from reviewers,
-	especially <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">W3C Horizontal Groups</a> [[CHARTER]]
-	and groups identified as dependencies in the charter
-	or identified as <a href="https://www.w3.org/2001/11/StdLiaison.html">liaisons</a> [[LIAISON]],
-	and seek evidence of clear communication to the general public
-	about appropriate times and which content to review
-	and whether such reviews actually occurred.
+	The objective of <dfn id=dfn-wide-review export>wide review</dfn> is to seek
+	feedback on a specification from the entire web community, including the general
+	public. The goal is to make the specification as robust as possible before it
+	becomes a [=W3C Recommendation=]. 
 
-	For example,
-	inviting review of new or significantly revised sections published in Working Drafts,
-	and tracking those comments
-	and the [=Working Group=]'s responses,
-	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
-	as well as the general public,
-	especially those affected by this specification,
-	a proposal to enter [=Candidate Recommendation=]
-	(for example in approximately 28 days).
-	By contrast a generic statement in a document
-	requesting review at any time
-	is likely not to be considered as sufficient evidence
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
+	Wide review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
```

Comment:
It would be helpful to add here some words to the effect that WR comments received on a spec that is in CR or later should be noted and resolved where possible but that it could be reasonable for WGs to defer those to a future iteration of the specification itself. 

I say this because looking at the delta relative to the previous text here, there was a strong sense that WR feedback before CR is somehow weightier than WR feedback during CR or PR, or even Rec. This sense was borne out of the fact that WR must be demonstrated in order to transition to CR but further WR need not be solicited _after_ CR.

My concern here is that WGs should be allowed to make good progress through the maturity levels without being pushed into some kind of circular pattern whenever any comment is received on a spec, especially one that has already made it to CR.

Reply:
I agree we do not want specs being held up. Equally I'm not sure how to put something as nuanced as you suggest into the Process in a way that is useful - sometimes there will be things caught at CR that are a showstopper.

Based on suggestions elsewhere in this thread, the following was updated to include the "heads up" note:
>Horizontal Review Groups should be notified of all new and significantly revised sections of a specification that has already been reviewed. Note: failure to do so may result in the transition to Candidate Recommendation being delayed.>

Does that help with your concern? If not could you suggest some succinct words/placement that would?

Reply:
That's great, thank you @LJWatson .